packages/teams-js/src/internal/constants.ts

```diff
@@ -108,50 +110,26 @@ export const getMediaCallbackSupportVersion = '2.0.0';
  */
 export const scanBarCodeAPIMobileSupportVersion = '1.9.0';
 
+export const validDomainsFallback = validDomains;
```

Comment:
This is needed since importing validDomains directly from the json and using its value `validDomains.validOrigins` is soon to be deprecated. Might rename this variable `validDomainsJSON` though